### PR TITLE
CreateEventForm

### PR DESCRIPTION
**Pull request by CristobalMe**
## Description
This PR handles the Create Event action, and displays a form to enable users to input their event data.

## Milestones
11. Create events

## Resources
None

## Test Plan

Screenshot 2024-07-09 at 1 58 54 PM
Screenshot 2024-07-09 at 1 59 00 PM
Screenshot 2024-07-09 at 1 59 06 PM

### CODE REVIEW

File: frontend/src/components/userPage/CreateEventForm.jsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
+import React, { useState } from "react";
+import axios from "axios";
+
+const CreateEventForm = () => {
+  const url = import.meta.env.VITE_URL;
+  const [form, setForm] = useState(false);
```
Comment (toriqyanM) [inline frontend/src/components/userPage/CreateEventForm.jsx:6]: Let's have more descriptive name here. What does this boolean flag control exactly?

File: frontend/src/components/userPage/CreateEventForm.jsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+                        Longitude:
+                        </label>
+                        <input
+                        type="text"
+                        className="border rounded w-full py-2 px-3 text-gray-700 leading-tight"
+                        value={long}
+                        onChange={(e) => setLong(e.target.value)}
+                        required
+                        />
+                    </div>
```
Comment (toriqyanM) [inline frontend/src/components/userPage/CreateEventForm.jsx:127]: So user needs to manually fill in latitude and longitude?

Comment (CristobalMe) [reply frontend/src/components/userPage/CreateEventForm.jsx:127]: Currently, yes.
**_To Do:_**
Implement a system to get long and lat

File: frontend/src/components/userPage/CreateEventForm.jsx (hunk 1 above)
Comment (CristobalMe) [reply frontend/src/components/userPage/CreateEventForm.jsx:6]: If the form is displayed

Comment (CristobalMe) [reply frontend/src/components/userPage/CreateEventForm.jsx:6]: **_To Do:_**
More descriptive name (form)

**Pull request merged**